src/com/griddynamics/main/Ticket.java

```diff
@@ -0,0 +1,41 @@
+package com.griddynamics.main;
```

Comment:
@kolegova-mary I suggest correcting our packages structure a bit. Example:
<img width="245" alt="Screenshot 2021-03-18 at 10 26 48 PM" src="https://user-images.githubusercontent.com/21138609/111685659-3e059a00-8839-11eb-9c28-b49333763f3a.png">
It will look clear and convenient for support

Reply:
other stuff looks good!